code/project_euler_problem19.java

```diff
@@ -0,0 +1,91 @@
+import java.util.HashMap;
+import java.util.Map;
+ 
+ 
+public class Problem_19 {
+    public static void main(String args[]){
+        Map<Integer, String> L = new HashMap<Integer, String>();
+        L.put(1, "Sun");
+        L.put(2, "Mon");
+        L.put(3, "Tue");
+        L.put(4, "Wed");
+        L.put(5, "Thurs");
+        L.put(6, "Fri");
+        L.put(7, "Sat");
+         
+        // Start at Monday, 1st, 1900
+        int counter = 1;
+        int tally = 0;
+         
+        // Year
+        for (int yr = 1900; yr < 2001; yr ++)
```

Comment:
Change all of the post increment to pre increment, and remove the space between operator and variable (`yr ++` => `++yr`).

Reply:
code is working in post increment form as well as pre increment form.

Reply:
@AdiChat please review.

Reply:
Using precinrement avoids an unnecessary copy for variables (Maybe not here, but you never know). Just because something works doesn't mean it is the best example of it. Make those changes as @iattempt suggested.